app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java

```diff
@@ -66,22 +71,35 @@ public View getSpinnerView_DropDown(int position, View convertView, ViewGroup pa
         /*
         if None is the object, we can't use Mood to get the information because Mood doesn't have this moodType
          */
-        if(objects[position].equals("None")){
-            dropDownText.setText("None");
+        if(objects[position].equals( context.getResources().getString(R.string.filter_empty) )){
```

Comment:
Unintended whitespace change?

```suggestion
        if(objects[position].equals(context.getResources().getString(R.string.filter_empty) )){
```